### DISCUSSION TITLE
Welcome to Pi-CryptoConnect Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/Pi-CryptoConnect#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by erictranvn** [comment]
Really?

**Comment by thinh383** [comment]
Are your projects really for the Pinetwork project?

**Reply by erictranvn** [reply]
I think it's fake .

**Reply by thinh383** [reply]
Vậy nếu là bạn liệu bạn có rảnh làm chơi mấy công việc này không?

**Reply by 3LABR8** [reply]
Well he did write some vode for pi network if you looked deep enough.  Youdont have to be a core team member to help out the ecosystem. Hes making it easir for the core team to implement services instead of them taking longer, they can focus on open mainnet pre requisites

**Reply by 3LABR8** [reply]
As a community we all do our part to make pi network apart of our daily lives

**Comment by Conghau295** [comment]
Is it real to have a pi value equal to 314,159$ and be activated open mainnet on 1-6-2024?

**Reply by Toilanguyenwang** [reply]
Tôi nghĩ là ko nhé

**Reply by Conghau295** [reply]
2-6-2024 sẽ rõ

**Comment by Ahmedfemo** [comment]
Hey kosasih
Iam femo , a web developer from egypt
And iam a pi pioneer 

Iam wondering of pi price and pi lunching date.

1.Can you tell me if pi price 314.159 usd
Is the official price or just a test?

2.And tell me the lunching dat 1 june is officially or just a test?

Best regard

**Reply by 3LABR8** [reply]
Its code. Pi coin is what we make of it

**Comment by tsotsoobi** [comment]
Does this give lead to Pi Network's own exchange or Dex?
